tutorials/R/Lidar/intro-to-lidar/what-CHM-DSM-DTM/what-CHM-DSM-DTM.md

```diff
@@ -18,26 +18,26 @@ urlTitle: chm-dsm-dtm-gridded-lidar-data
 
 ## Common LiDAR Data Products
```

Comment:
After looking at this tutorial for a bit, I would like to suggest a re-arrange of the top section. What do you think of this order: 

1. LIDAR point clouds:  move this to the top
2. Common LIDAR data products: Keep the definition statements here, but remove the links to the SJER_--M_3d.html models (we'll use those later). Let's use the cartoons to explain the concepts at first (take a look at the graphics I added links to below). Links and graphics of SJER 3d models will move below the free software links (4) below. 
3. Free point-cloud viewers: list the three here, and also mention that there are useful pointcloud viewers available through Rstudio as demonstrated in our `<a href="https://www.neonscience.org/structural-diversity-discrete-return"> Structural Diversity tutorial </a>`
4. Add a new header here along the lines of "## 3D Models of NEON site: SJER" 
  a. Include a bit of text, "Click the graphics below to view interactive 3d models of our `<a href="https://www.neonscience.org/field-sites/field-sites-map/SJER">San Juaquin Experimental Range site.</a>`
  b. Then post the graphics that link to the SJER_--M_3d.html models

... "Gridded, or Raster, LiDAR Data Products" and below looks ok to me.